clients/js/src/generated/programs/token2022.ts

```diff
@@ -223,6 +225,9 @@ export function identifyToken2022Instruction(
   if (containsBytes(data, getU8Encoder().encode(11), 0)) {
     return Token2022Instruction.ThawAccount;
   }
+  if (containsBytes(data, getU8Encoder().encode(38), 0)) {
+    return Token2022Instruction.WithdrawExcessLamports;
+  }
   if (containsBytes(data, getU8Encoder().encode(12), 0)) {
     return Token2022Instruction.TransferChecked;
   }
```

Comment:
Could you make sure you add the new instruction in the `idl.json` in a way that keeps the discriminators ordered? This makes it easier to find them in the future. I.e., here instruction of discriminator 38 should not be between 11 and 12.